guidelines/templates/resources/configuration.md

```diff
@@ -0,0 +1,28 @@
+---
+title: {Component name} {sub-}chart
+type: Configuration
+---
+
+>**NOTE:** Blockquotes in this document provide instructions. Remove them from the final document.
+>
+>This document is a ready-to-use template for the **Configuration** document type that provides technical details about configuration of a Kyma chart or sub-chart. Follow the `05-{00}-{component-name}.md` convention to name the document. Make sure that the configuration document for the main chart displays first on the list. If there is only one document of a certain type, remove the `type` metadata completely so that the document displays well on the UI. 
+
+To configure the {Component name} {sub-}chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.
+
+>Use the tip feature to provide links to the documentation about Kyma overrides. Add also a link to examples of either top-level charts overrides for charts configuration documents or sub-charts overrides for sub-chart configuration documents.
+
+>**TIP:** To learn more about how to use overrides in Kyma, see the following documents: 
+>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
+>* {[Top-level charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-top-level-charts-overrides)}
+>* {[Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)}
+
+## Configurable parameters
+
+This table lists the configurable parameters, their descriptions, and default values:
+
+| Parameter | Description | Default value |
+|-----------|-------------|---------------|
+| **{parameter}** | {Parameter description} | `{parameter value}` |
+| **{another.parameter}** | {Another parameter description} | `{another parameter value}` |
```

Comment:
I am thinking if repeating it is required? 🤔

Reply:
Removed :)